modules/cnf-performance-profile-creator-arguments.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+// Epic CNF-792 (4.8)
+// * scalability_and_performance/cnf-create-performance-profiles.adoc
+
+
+[id="performance-profile-creator-arguments_{context}"]
+=  Performance Profile Creator arguments
+
+.Performance Profile Creator arguments
+["cols="30%,5%,65%",options="header"]
+|===
+| Argument | Values | Description
+
+| --mcp-name
+a| worker-cnf (default)
```

Comment:
Let's not specify the default value for the "required" fields, it seems a little odd, right? We can add an "example" instead